client/src/endpoint/account.rs

```diff
@@ -410,11 +410,11 @@ impl IntoRequest for Effects {
     }
 }
 
-// impl Cursor<Effect> for Effects {
-//     fn cursor(self, cursor: &str) -> Self {
-//         self.cursor(cursor)
-//     }
-// }
```

Comment:
this was commented out copy pasta... do we need it?

Reply:
Yes, the trait implementation is necessary for using the endpoint in an iterator.